src/main/java/hunternif/mc/atlas/RegistrarAntiqueAtlas.java

```diff
@@ -0,0 +1,65 @@
+package hunternif.mc.atlas;
+
+import hunternif.mc.atlas.item.ItemAtlas;
+import hunternif.mc.atlas.item.ItemEmptyAtlas;
+import hunternif.mc.atlas.item.RecipeAtlasCloning;
+import hunternif.mc.atlas.item.RecipeAtlasCombining;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.oredict.ShapelessOreRecipe;
+
+@Mod.EventBusSubscriber(modid = AntiqueAtlasMod.ID)
```

Comment:
Rather than registering for the bus, I'd check if the item is needed and listen to events on an instance subscriber (which will not get registered if the item is not needed) You can register this during construction.